packages/replay-internal/src/util/createPerformanceEntries.ts

```diff
@@ -175,38 +175,44 @@ function createResourceEntry(
  * Add a LCP event to the replay based on a LCP metric.
  */
 export function getLargestContentfulPaint(metric: Metric): ReplayPerformanceEntry<WebVitalData> {
-  return getWebVital(metric, 'largest-contentful-paint');
+  const lastEntry = metric.entries[metric.entries.length - 1] as (PerformanceEntry & { element?: Element }) | undefined;
+  const element = lastEntry ? lastEntry.element : undefined;
+  return getWebVital(metric, 'largest-contentful-paint', element);
 }
 
 /**
  * Add a CLS event to the replay based on a CLS metric.
  */
 export function getCumulativeLayoutShift(metric: Metric): ReplayPerformanceEntry<WebVitalData> {
-  return getWebVital(metric, 'cumulative-layout-shift');
+  return getWebVital(metric, 'cumulative-layout-shift', undefined);
```

Comment:
Is there not an attribution element for CLS? I see something like this [in the web-vitals lib](https://github.com/GoogleChrome/web-vitals/blob/21c16dd3400c0f598d69af474234a5d025f6ee8f/src/types.ts#L69-L80)

Reply:
oh there is, let me add that too